pkg/kubelet/server/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- tallclair
```

Comment:
anyone else from pkg/kubelet/OWNERS approvers that should be hoisted here? I think this means you'll be targeted specifically before bubbling up

Reply:
Good to know.

@cheftako I'm nominating you to be the second reviewer for the kubelet server. LMK if you don't want it, or want to nominate anyone else.